cmd/goordinator/main.go

```diff
@@ -280,6 +287,65 @@ func hide(in string) string {
 	return "**hidden**"
 }
 
+func startPullRequestAutoupdater(config *cfg.Config, githubClient *githubclt.Client, mux *http.ServeMux) (*autoupdate.Autoupdater, chan<- *github.Event) {
+	if !config.Autoupdater.TriggerOnAutoMerge && len(config.Autoupdater.Labels) == 0 {
+		logger.Info(
+			"autoupdater is disabled, trigger_on_auto_merge is false and trigger_labels in config is empty",
+			logfields.Event("autoupdater_disabled"),
+		)
+
+		return nil, nil
+	}
+
+	if len(config.GithubAPIToken) == 0 {
+		fmt.Fprintf(os.Stderr, "ERROR: config file %s: github_api_token must be provided when autoupdater is enabled", *args.ConfigFile)
```

Comment:
Nitpick and optional ofc, but i suppose you could export constants from the config pkg that hold the key names, in this case `github_api_token`, same for all other configuration keys.

Reply:
good idea, fits better to define them in the cfg package.
The whole process of validating the configuration file could be even move into the cfg package.

I'll do that in the future